src/main/scala/io/iohk/ethereum/jsonrpc/server/http/JsonRpcHttpServer.scala

```diff
@@ -31,6 +36,16 @@ trait JsonRpcHttpServer extends Json4sSupport {
 
   def corsAllowedOrigins: HttpOriginMatcher
 
+  val minRequestInterval: FiniteDuration
```

Comment:
We can add in JsonRpcHttpServer the val config: JsonRpcHttpServerConfig.
So, in this way, we don't need the next params:
 
 
-   val minRequestInterval: FiniteDuration
-   val latestTimestampCacheSize: Int
-   val latestRequestTimestamps: LruMap[RemoteAddress, Long]
-   val ipTrackingEnabled: Boolean
 
Example:
 
trait JsonRpcHttpServer extends Json4sSupport {
  val jsonRpcController: JsonRpcBaseController
  val jsonRpcHealthChecker: JsonRpcHealthChecker
  val config: JsonRpcHttpServerConfig
 
And we access these values through the config.